index.d.ts

```diff
@@ -0,0 +1,19 @@
+export interface FileMatcherOptionsI {
+    path?: string;
+    recursiveDepth?: number;
+    readFileConcurrency?: number;
+    micromatchOptions?: any;
+    filter?: {
+      namePatterns?: string[];
+      contentRegExp?: string;
+      readOptions?: string;

```

Comment:
Thank you for the PR :)
- `readOptions` is not a string but an object. Adjust to `any` ?
- `contentRegExp` should be a regular expression. Maybe `RegExp` is better

Reply:
Done.